### PR TITLE
Issue/42/initialization

### PR DESCRIPTION
**Pull request by chto**
I have implemented a method to include prior information into our plots. 
I also solve the flip sign problem between TDC2 and PyCS. For convenience, I adopt the convention defined in PyCS which means T_AB = T_A-T_B.

### CODE REVIEW

**Comment by chto** [comment]
Hi @drphilmarshall , 
I have implemented a method to include prior information into our plots.
I also solve the flip a sign problem for time delay by adopting convention in PyCS team. 
Besides, I also implement a sigma_intrinsic to describe the extra noise. 
Could you review my code? Thanks.

**Comment by chto** [comment]
Btw, It should also be able to merge to master branch. I solve the merge problem in my local machine.

**Comment by drphilmarshall** [comment]
Great stuff! I'll review this on Thursday, when I am back.

**Comment by chto** [comment]
Hi @drphilmarshall,
The code is ready for review.

File: python/desc/sltimer/reading.py [hunk 1]
```diff
@@ -318,6 +318,55 @@ def select_bands(lcs, bands):
...
+            # Now find the overall mean, and compute each filter's offsets:
+            overall_mean = np.mean(lcs[j].mags[season.indices])
+
+            offset = dict()
+            for band in bands:
+                offset[band] = indiv_mean[band] - overall_mean
+
+            # Finally, apply the offsets:
+            for band in bands:
+                lcs[j].mags[np.intersect1d(index[band], season.indices)] -= offset[band]
```
Comment (drphilmarshall) [inline python/desc/sltimer/reading.py:370]: Here, are you subtracting the mean magnitude or the mean flux? It looks like the mean magnitude. That's OK, we just have to be very clear about what we are doing, and what it means. Can you make sure the docstring of this method is correct, and as full as possible, please?

**Review by drphilmarshall — CHANGES_REQUESTED** [review]
Some concern about new whitening, and whether it acts on magnitudes or fluxes...

**Comment by drphilmarshall** [comment]
I think we could merge this, after you tracked down the problem and switched to investigating $\sigma_{intrinsic}$. I think we will soon want to move the notebook you finished here into some sort of "attic" folder, and list the results with the better model as the package demo. We can do that later, though. For now, our coverage is failing! That means we are not writing enough unit tests. Can you think of some good ones based on your experience so far? 

> A good way to program is to write tests that would re-create bugs, if you hadn't fixed the code already. (Likewise, whenever you get a bug, first write the unit test that fails because of the bug, and then fix the code until the test passes.)

**Comment by drphilmarshall** [comment]
Hey @chto - can you please turn off the stdout? It's making the notebooks very very long, and delaying the enjoyment of your new intrinsic sigma analysis plot :-)

**Comment by drphilmarshall** [comment]
Hi @chto - I think you have moved well beyond the initialization problem now, so I have merged these commits into `issue/40/tdc2gatewaytest`, which is where we should write our LSST DESC Note on your and Milan's work.

**Pull request closed without merging**